### DISCUSSION TITLE
Problems with installation due to gconf-service and gconf2 deprecated in ubuntu 23.10

### DISCUSSION THREAD

**Discussion by SagieSH** [Eugeny/tabby#9456] [General]
When I attempt to install the software, I get the following error:
![image](https://github.com/Eugeny/tabby/assets/72133323/056e4429-0ffa-401d-8426-c6200437a7c7)
And after some research I found that this is due to deprecation of these packages.
Is there a way to install this anyway? apt install for gconf2 and gconf-service doesn't work....

**Comment by teicors** [comment]
Hi, 
I solved this issue with
wget http://ftp.it.debian.org/debian/pool/main/g/gconf/gconf2_3.2.6-8_amd64.deb
wget http://ftp.it.debian.org/debian/pool/main/g/gconf/libgconf-2-4_3.2.6-8_amd64.deb
wget http://ftp.it.debian.org/debian/pool/main/g/gconf/gconf2-common_3.2.6-8_all.deb
wget http://ftp.it.debian.org/debian/pool/main/g/gconf/gconf-service_3.2.6-8_amd64.deb
wget http://ftp.it.debian.org/debian/pool/main/o/openldap/libldap-2.5-0_2.5.13%2bdfsg-5_amd64.deb
and 
dpkg -i *.deb
under ubuntu 23.10

BR,
Angelo

**Reply by ShadowDev7** [reply]
Thank you bro. You fixed it

**Reply by Gasparonik** [reply]
Thank you bro!
